R/custom_deconvolution_methods.R

```diff
@@ -29,7 +29,7 @@ custom_deconvolution_methods <- c(
   "EPIC" = "epic",
   "CIBERSORT" = "cibersort",
   "ConsensusTME" = "consensus_tme",
-  "BASE" = "base"
+  "seqImmuCC" = "seqimmucc"
```

Comment:
What happened to BASE?

Reply:
Removing a method is a breaking change that should only happen with a good reason (+ major version bump + explanation in changelog)

Reply:
I'm not removing a method here, I'm just removing the possibility to use it with a custom matrix (this is due to an error in my implementation)

Reply:
all good then